### PR TITLE
Medical core

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
Deployed Link: [Medical Core](https://635641c53255bf32cb840c06--peppy-otter-7e90dc.netlify.app)

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+.main-section__button {
+    cursor: pointer;
+    color: #1BA0AB;
+    background: #FFF5F0;
+    font-family: 'Montserrat', sans-serif;
+    padding: .8rem 1.5rem;
+    border-radius: .5rem;
+}
+
+.main-section__button__primary {
```
Comment (Morrism1) [inline styles/styles.css:110]: Also here, 2 nested elements

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    background: #1BA0AB;
+    color: #fff;
+    border: none;
+    cursor: pointer;
+    font-family: 'Montserrat', sans-serif;
+    padding: .8rem 1.5rem;
+    border-radius: .5rem;
+}
+
+.main-section__button__secondary {
```
Comment (Morrism1) [inline styles/styles.css:120]: same thing here.

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+    margin-right: 1rem;
+    cursor: pointer;
+    color: #1BA0AB;
+    background: #FFF5F0;
+    font-family: 'Montserrat', sans-serif;
+    padding: .8rem 1.5rem;
+    border-radius: .5rem;
+}
+
+.main-section__button__primary--is-hovered {
```
Comment (Morrism1) [inline styles/styles.css:131]: Also here!

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+    display: none;
+}
+
+.main-section__button__primary:hover>.main-section__button__primary--is-hovered {
```
Comment (Morrism1) [inline styles/styles.css:135]: 2 issues here, they way you are targeting element and 2 nested elements. One unique class would do using BEM

File: styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+.card__media {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    font-size: .5rem;
+}
+
+.card__media img {
```
Comment (Morrism1) [inline styles/styles.css:253]: This way of targeting element is invalid in BEM.

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+.footer__form {
+    margin: 4rem 0;
+    display: flex;
+}
+
+.footer__input {
+    flex-grow: 1;
+}
+
+.footer__input input {
```
Comment (Morrism1) [inline styles/styles.css:301]: This too.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+    <header class="header">
+        <h1 class="header__title">Medical Core</h1>
+        <ul class="header__list">
+            <li>Home</li>
+            <li>Product</li>
+            <li>Pricing</li>
+            <li>Contact</li>
+        </ul>
+        <div class="header__actions">
+            <button type="button" class="header__button__tertiary">Login</button>
```
Comment (Morrism1) [inline index.html:23]: `header__button__tertiary` is not a valid BEM class. Two Elements on the same class is not allowed.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+            <button type="button" class="header__button__primary">JOIN US
```
Comment (Morrism1) [inline index.html:24]: `header__button__primary` is not a valid BEM class. Two Elements on the same class is not allowed.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+                <span class="header__button__primary--is-hovered"> <i class="fa fa-arrow-right"></i></span>
```
Comment (Morrism1) [inline index.html:25]: `header__button__primary--is-hovered` is not a valid BEM class. Two Elements on the same class is not allowed.

**Review by Morrism1 — CHANGES_REQUESTED** [review]
There are things to change.

**Comment by Ntwali-Josue** [comment]
> There are things to change.

Thanks for the review, I am going to work on them.

File: styles/styles.css [hunk 10]
```diff
@@ -0,0 +1,321 @@
...
+    background: #1BA0AB;
+    color: #fff;
+    border: none;
+    cursor: pointer;
+    font-family: 'Montserrat', sans-serif;
+    padding: .8rem 1.5rem;
+    border-radius: .5rem;
+}
+
+.header__button__secondary {
```
[the remark this answers is not in the document]
Comment (Morrism1) [reply styles/styles.css:66]: Why did mark this as resolved while it is still here??

**Review by Morrism1 — CHANGES_REQUESTED** [review]
One thing you missed

**Review by Morrism1 — APPROVED** [review]
🚀🚀

**Pull request merged**